webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/index.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Grid, GridItem } from '@patternfly/react-core';
+
+import Properties from '../../Properties';
+import ParametersCard from '../../Parameters';
+import InterfacesCard from '../../Interfaces';
+import AuditCard from '../../Audits';
+import StatusAlert from '../../Status';
+import './Details.css';
+
+const DetailsTab = ({ response }) => (
+  <div className="details-tab">
+    <Grid>
+      <GridItem offset={3} span={4}>
+        <StatusAlert status={response ? response.global_status_label : null} />
+      </GridItem>
+    </Grid>
+    <Grid className="details-cards">
+      <GridItem span={3} rowSpan={3}>
+        <Properties hostData={response} />
+      </GridItem>
+      <GridItem style={{ marginLeft: '40px' }} span={3}>
```

Comment:
Should we rather use stylesheets instead of inline style rules?

Reply:
we should, and this already done by the follow up [PR](https://github.com/theforeman/foreman/pull/8086/files#diff-4d352358ba0d864a6da40d61668f0741dfd2342e51c0d28be9f37c6c569a2dd0R13) which fixes the grid layout